lstchain/datachecks/containers.py

```diff
@@ -130,13 +140,28 @@ def fill_event_wise_info(self, subrun_index, table, mask, geom,
             # of every subsequent subrun file (if analyzed on its own) will have ucts_jump=True, 
             # but these are not new jumps, just the ones from previous subruns, so they should 
             # not be counted.
-            uj = table['ucts_jump']
+            uj = table['ucts_jump'].to_numpy(copy=True)
             # find the first False value, and set to False also all the earlier ones:
             first_non_jump = np.where(uj==False)[0][0]
             uj[:first_non_jump] = False
+            # count only the jumps occurring in events of the type we are
+            # processing:
             self.num_ucts_jumps = np.sum(uj[mask])
-        self.mean_alt_tel = np.mean(table['alt_tel'])
-        self.mean_az_tel = np.mean(table['az_tel'])
+
+        # since azimuth can go through 0, just take the pointing of the
+        # event in the middle of the table (the actual mean value would be
+        # problematic for culmination towards north, az= ~0  ~2pi):
+        self.mean_az_tel = table['az_tel'][int(len(table)/2)]
+        self.mean_alt_tel = table['alt_tel'][int(len(table)/2)]
+        time_utc = Time(table['dragon_time'][int(len(table)/2)],
+                        format="unix", scale="utc")
+        # Calculate telescope pointing in sky coordinates
+        telescope_pointing = SkyCoord(alt=self.mean_alt_tel*u.rad,
+                                      az=self.mean_az_tel*u.rad,
+                                      frame=AltAz(obstime=time_utc,
+                                                  location=location))
+        self.tel_ra = telescope_pointing.icrs.ra.to(u.deg).value
```

Comment:
`to_value(u.deg)`

Reply:
Why strip the unit away at all?

Reply:
Also, this does the altaz → icrs transformation twice I believe. Transform once, than access the result.

I would probably only store `self.pointing_icrs = self.pointing_altaz.transform_to('icrs')` and not the individual values without units.